runtime/compiler/env/J9PersistentInfo.hpp

```diff
@@ -63,11 +58,16 @@ enum IprofilerStates {
    IPROFILING_STATE_OFF,
 };
 
-enum JITaaSModes {
-   NONJITaaS_MODE = 0,
-   CLIENT_MODE,
-   SERVER_MODE,
+#if defined(JITSERVER_SUPPORT)
+namespace JITServer
+{
+enum RemoteCompilationModes {
```

Comment:
Move the brace on the next line and indent it by 3 spaces

Reply:
Will do. `RemoteCompilationModes` will have different style from the other enums defined in this file though, such as IprofilerStates

```
enum IprofilerStates {
   IPROFILING_STATE_ON = 1,
   IPROFILING_STATE_GOING_OFF,
   IPROFILING_STATE_OFF,
};
```